Src/Coravel/Queuing/HostedService/QueuingHost.cs

```diff
@@ -16,14 +16,16 @@ internal class QueuingHost : IHostedService, IDisposable
         private Queue _queue;
         private IConfiguration _configuration;
         private ILogger<QueuingHost> _logger;
+        private QueueOptions _queueOptions;
         private readonly string QueueRunningMessage = "Coravel Queuing service is attempting to close but the queue is still running." +
                                                       " App closing (in background) will be prevented until dequeued tasks are completed.";
 
-        public QueuingHost(IQueue queue, IConfiguration configuration, ILogger<QueuingHost> logger)
+        public QueuingHost(IQueue queue, IConfiguration configuration, ILogger<QueuingHost> logger, QueueOptions queueOptions)
         {
             this._configuration = configuration;
             this._queue = queue as Queue;
             this._logger = logger;
+            this._queueOptions = queueOptions;
```

Comment:
You have both IConfiguration and QueueOptions here, why not check the appSettings.json value here and  determine which one should be kept?